src/main/java/be/acara/frontend/service/OrderServiceImpl.java

```diff
@@ -0,0 +1,48 @@
+package be.acara.frontend.service;
+
+import be.acara.frontend.controller.dto.CreateOrderDto;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.stereotype.Service;
+
+@Service
+public class OrderServiceImpl implements OrderService {
+
+
+    private final OrderFeignClient orderFeignClient;
+
+    @Autowired
+    public OrderServiceImpl(OrderFeignClient orderFeignClient) {
+        this.orderFeignClient = orderFeignClient;
+    }
+
+
+    @Override
+    public void create(Long eventId) {
+        orderFeignClient.create(CreateOrderDto.builder()
+                .eventId(eventId)
+                .amountOfTickets(1)
+                .build());
+    }
```

Comment:
Considering OrderFeignClient.create had ReponseEntity<Void> as type, I had expected a check on the creation status through HttpStatus. 🤔